File: readJSON.py
import json

# Open the JSON file
with open('books.json') as file:
    data = json.load(file)

book1 = data ['Books'][0]
book2 = data ['Books'][1]

print("Title of the first book: ", book1['title'])
print("Author of the second book: ", book2['author'])
print("Genres of all books")

for index, book in enumerate(data['Books']):
    print(f"Book {index + 1} Genres:", ", ".join(book['genres']))

File: readJSON.js
const fs = require('fs');

fs.readFile('books.json', 'utf8', (err, data) => {
  if (err) {
    console.error("Error reading the file:", err);
    return;
  }

  const booksData = JSON.parse(data);

    console.log("Title of the first book", booksData.Books[0].title);
   
    console.log("Author of the second book: ", booksData.Books[1].author);
   
    console.log("Genres of all books: ");
    booksData.Books.forEach((book,index) => {
      console.log(`Book ${index + 1} Genres:`, book.genres.join(", "));
        
    });

    
  });